pkg/util/clihelper/clihelper.go

```diff
@@ -144,8 +146,9 @@ func HasPodProblem(pod *corev1.Pod) bool {
 }
 
 func CheckHelmVersion(output string) error {
-	if !(strings.Contains(output, "Version:\"v3.")) {
-		return errors.New("Please ensure that the \"helm\" binary in your PATH is valid. Only Helm v3 is supported")
+	min := "v3.10.0"
```

Comment:
Can we make this a public const?

Reply:
Sure, in this package here or rather in the helm package?

Reply:
I think it's fine to keep it here, given that it'll be used in this package.
Can always refactor it later on, if it makes more sense somewhere else